spec/models/tweet_spec.rb

```diff
@@ -1,17 +1,15 @@
-require 'spec_helper'
+# frozen_string_literal: true
 
 describe Tweet do
-  fixtures :affiliates
-
-  before do
-    allow(Twitter).to receive(:user).and_return double('Twitter', id: 12345, name: 'USASearch', profile_image_url: 'http://some.gov/url')
```

Comment:
I removed this `allow(Twitter).to receive(:user)`... line, as it was related to code that is no longer used.